DESY/VBF_M125_CP_mt_v1.txt

```diff
@@ -0,0 +1,25 @@
+
+=========
+   v1
+=========
+
+* Year: 2017
+
+* Base sample:
+/VBFHToTauTau_M125_13TeV_powheg_pythia8/RunIIFall17MiniAODv2-PU2017_12Apr2018_new_pmx_94X_mc2017_realistic_v14-v1/MINIAODSIM
+
+* Path to Synchronisation Ntuples: 
+/afs/desy.de/user/f/filatovo/public/CP_Synch_2017/VBF_HToTauTau_M125_CP_mt_v1.root
+
+* Selection criteria and cuts follow the ones from https://twiki.cern.ch/twiki/bin/viewauth/CMS/HiggsCPinTauDecaysSync
+* Corrections are taken from IC workspace      
+* Comments:
+  - BTagAlgorithm: DeepCSV, Medium WP with promote-demote SF corrections applied
+  - PUPPI MET recoil corrections are not yet applied (while PFMET is corrected according to all the recommendations)
+  - SVFit and FastMTT variables are calculated with PFMET
+  - lep->tau fakes and TES are currently taken for an old MVA since official TAU POG recommendations are not available for DeepTau
+  - Tau Xtrigger: hltSelectedPFTau27LooseChargedIsolationAgainstMuonL1HLTMatched is missing, hltPFTau27TrackLooseChargedIsoAgainstMuon used instead
+  - TauSpinner weights: different naming, stored in GenTauCheck tree
```

Comment:
It is not possible to access different tree and branches with different names with tools for automated comparison of sync ntuples. So, it is welcome that all stuff is stored in the same tree using standard branch names.

Reply:
No problem, I will change the naming and location of branches in the following update. Still, this affects only TauSpinner weights, which are trivial for the case of VBF sample. The rest of the variables is stored in a single TauCheck tree and is named according to the CP Sync twiki page, so in principle there should be no problem for an automated comparison of the key variables.